ci-operator/config/openshift/microshift/openshift-microshift-release-4.13.yaml

```diff
@@ -97,9 +97,18 @@ tag_specification:
 tests:
 - as: verify
   cluster: build02
-  commands: make verify
-  container:
-    from: src
+  steps:
+    test:
+    - as: verify
+      commands: make verify-ci
```

Comment:
Keep in mind that new static analysis only runs on 4.14+. This target does not exist in this branch.